.github/ISSUE_TEMPLATE/Service_team_request.md

```diff
@@ -3,7 +3,7 @@ name: "\U0001F6A7Service Team Support Request"
 about: For select service teams to request CLI-team support for work such as new features,
   commands, etc.
 title: ''
-labels: ''
+labels: Service Team Support Request
```

Comment:
Personally, I will just use `Feature Request` for two issue templates. But I'm ok with this one too.

Reply:
one step further, what's the action for "service team support request" issues? should we az cli team take action, or service team take action? If you really want to add this label, I would suggest to separate feature request to az cli team, and to service team (for their owned cli/extension).

Reply:
@yungezz currrently 

- This PR only adds first level labels (Feature Request for CLI self and Feature Request to support new module/command) automatically when creating an issue based on issue template. And the following optimization is using a bot to extract issue title/description to add second level (service level) labels (such as network, storage, compute and etc.) automatically.
- Service Team Support Request means service teams to request CLI-team support for work such as new features, commands, etc. Usually CLI team need take action: add new features/commands on behalf of service team if CLI team owns the service, or provide some guidance for service team if service team need adds features/commands themselves if CLI team doesn't own the service.